src/components/Header/index.less

```diff
@@ -50,22 +44,82 @@
       .react-geo-nominatimsearch.ant-select-auto-complete {
         width: 60%;
       }
-
-      @media screen and (max-width: 680px) {
-        .react-geo-nominatimsearch.ant-select-auto-complete {
-          width: 90%;
-        }
-      }
     }
+
     &.right-items {
       flex: 1;
       justify-content: flex-end;
     }
+  }
+
+  @media (max-width:599px) {
```

Comment:
is this for smartphones or tablets or both? I think a short comment explaining that would be useful here